src/services/Wrapper.js

```diff
@@ -167,9 +166,6 @@ export default class Wrapper {
     });
   }
   async performEDCDataTransfer(assetId, providerConnector, requestHeaders) {
-    if((BACKEND === 'true') || (BACKEND == true)){
-      return await backendService.getPassportV1(assetId);
```

Comment:
Moved logic to PassportView, because it was giving a exception in the wrapper.js